cli/tfsource/types.go

```diff
@@ -43,9 +45,23 @@ func (src *TerraformSource) String() string {
 // so the same file path (/foo/bar) is always considered the same version. See also the encodeSourceName and
 // ProcessTerraformSource methods.
 func (terraformSource TerraformSource) EncodeSourceVersion() string {
+	if IsLocalSource(terraformSource.CanonicalSourceURL) {
+		hash, err := terraformSource.HashLocalDir()
```

Comment:
How would you want to handle the error?
Changed it now to keep the default source version generation strategy in case of error which would result in the same behaviour as it is currently.